data/papers.yaml

```diff
@@ -1,3 +1,14 @@
+- title: "Want More Unikernels? Inflate Them!"
+  venue: ACM SoCC'22
+  year: 2022
+  pdf: https://dl.acm.org/doi/abs/10.1145/3542929.3563473
```

Comment:
```suggestion
  link: https://dl.acm.org/doi/abs/10.1145/3542929.3563473
  pdf: https://dl.acm.org/doi/pdf/10.1145/3542929.3563473
```

Reply:
Thanks @razvand. We can commit with your changes.